service/worker/scheduler/workflow.go

```diff
@@ -144,6 +145,9 @@ type (
 		// in a single SideEffect
 		nextTimeCacheV1 map[time.Time]getNextTimeResult
 		nextTimeCacheV2 *schedspb.NextTimeCache
+
+		visibilityManager       manager.VisibilityManager
+		searchAttributeProvider searchattribute.Provider
```

Comment:
what are these for? we shouldn't be accessing this stuff from the workflow (or the activities either I think)

Reply:
Forgot to remove.